libs/MVS/Image.h

```diff
@@ -119,21 +120,28 @@ class MVS_API Image
 	#ifdef _USE_BOOST
 	// implement BOOST serialization
 	template<class Archive>
-	void save(Archive& ar, const unsigned int /*version*/) const {
+	void save(Archive& ar, const unsigned int version) const {
 		ar & platformID;
 		ar & cameraID;
 		ar & poseID;
+		if (version > 2) {
```

Comment:
this is not enough, you need to inform boost of the current version, ex:
```
BOOST_CLASS_VERSION(MVS::Image, 3)
```
Also, current version is 0, so no need to set it to 3, as 1 is enough.

Pls note that `Interface.h` does not use boost, but a custom serialization in order to be portable. So do not confuse the 2, no connection between them.

Reply:
thanks a lot! it should be resolved in the new commit.